src/utils/ADempiere/valueFormat.js

```diff
@@ -161,97 +163,135 @@ export function formatDate(date, isTime = false) {
   }))
 }
 
-//  Get Formatted Price
-export function formatPrice(number, currency) {
-  if (this.isEmptyValue(number)) {
-    return undefined
+/**
+ * Get formatted price and show currency
+ * @param {number} value
+ * @param {string} currency default 'USD'
+ * @returns {string} number format with thousands separator, precision, currency format
+ */
+export function formatPrice(value, currency, country) {
+  if (isEmptyValue(value)) {
+    value = 0
   }
-  if (this.isEmptyValue(currency)) {
+
+  if (isEmptyValue(currency)) {
     currency = getCurrency()
   }
-  //  Get formatted number
-  return new Intl.NumberFormat(getCountryCode(), {
+
+  // const precision = getStandardPrecision()
+
+  if (isEmptyValue(country)) {
+    country = getCountryCode()
```

Comment:
Don't touch it if not exists changes

Reply:
In some cases the country may change and the parameter can be used for this, if it is empty as it is by default the country code of the session is brought in.